database/migrations/2021_03_03_100000_create_alternate_emails_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAlternateEmailsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('alternate_emails', function (Blueprint $table) {
+            $table->id();
+            $table->string('email');
+            $table->foreignId('user_id')->constrained()->onDelete('cascade');
+            $table->timestamps();
+        });
+    }
+
+    public function down()
+    {
+        Schema::dropIfExists('alternate_emails');
+    }
```

Comment:
We don't need down methods for migrations. I never use them since `migrate:fresh` was introduced to Laravel.